packages/discord.js/src/structures/Guild.js

```diff
@@ -1365,7 +1365,9 @@ class Guild extends AnonymousGuild {
     const channelIsCategory = channel.type === ChannelType.GuildCategory;
     const types = getSortableGroupTypes(channel.type);
     return discordSort(
-      this.channels.cache.filter(c => types.includes(c.type) && (channelIsCategory || c.parentId === channel.parentId)),
+      this.channels.cache.filter(
+        ({ parentId, type }) => types.includes(type) && (channelIsCategory || parentId === channel.parentId),
```

Comment:
Why destructure just here and not in other places?

Reply:
I've tried to keep it consistent and simple but in some places, the preferred name is already declared so I had to work around it